### DISCUSSION TITLE
error packages anomalize

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#859] [Q&A] [answered]
Dear rekan2 Algoritma,

Mohon bantuannya,

Saya jalankan syntax anomalize hasilnya error seperti dibawah ini:

![image](https://github.com/teamalgoritma/community/assets/103427989/422eb3f5-74f2-4976-b7cd-622ba6d1bf8e)

Setelah dicek, pada proses time_decompose() nya jalan, hasilnya seperti ini:
![image](https://github.com/teamalgoritma/community/assets/103427989/a6153307-b8f1-4ccd-b2d5-2e8014777e3d)

Masalahnya ada pada saat jalankan anomalize(), cuma saya sudah ikutin sesuai dengan dokumentasinya masih error juga.

Mohon sarannya,

Best regards,
JT

**Comment by ramdhinafinita** [comment, marked as answer]
Dear @joejnt Terimakasih atas pertanyaanya. 

Error yang muncul bisa disebabkan karena parameter setelah remainder yang tidak digunakan. Bisa dicoba terlebih dahulu untuk methode anlomalize hanya menggunakan paramater remainder saja dulu. 

Jika masih terkendala error, bisa di uninstall dan install ulang packages nya dan setelah itu dengan menggunakan dataset yang ada di [tutorial berikut ](https://cran.r-project.org/web/packages/anomalize/vignettes/anomalize_quick_start_guide.html) untuk mencoba dan cek apakah packages anomalize nya berfungsi kembali.

Serta bisa di cek kembali parameter yang digunakan untuk anomalize apakah sudah cocok sesuai dengan yang diminta. https://cran.r-project.org/web/packages/anomalize/anomalize.pdf


Terimakasih, semoga membantu
